packages/atlas/src/hooks/useNftState.ts

```diff
@@ -0,0 +1,65 @@
+import { useMemo } from 'react'
+
+import { useNft } from '@/api/hooks'
+import { useUser } from '@/providers/user'
+
+export const useNftState = (id?: string) => {
```

Comment:
I think this could accept the nft data directly, the user of this hook should be responsible for fetching it already